### PR TITLE
Long pdf clip message

### PR DESCRIPTION
**Pull request by mttwc**
Putting this PR out there, _though this has not been styled to redlines_.

The Clipping Pdf progress panel now shows a message after 5s of delay. Added sinon faketimers tests.

### CODE REVIEW

File: src/scripts/clipperUI/mainController.tsx [hunk 1]
```diff
@@ -256,6 +257,10 @@ export class MainControllerClass extends ComponentBase<MainControllerState, Main
 			case PanelType.RegionInstructions:
 				return <RegionSelectingPanel clipperState={this.props.clipperState} />;
 			case PanelType.ClippingToApi:
+				if (this.props.clipperState.currentMode.get() === ClipMode.Pdf) {
+					return <ClippingPanelWithDelayedMessage clipperState={this.props.clipperState}
```
Comment (leakymemory) [inline src/scripts/clipperUI/mainController.tsx:261]: We should do this as a separate change, but I'd like to add a more generic version of this for any of the non-PDF clips that take more than a few seconds to come back.  Filed #116 to track.

**Review by leakymemory — APPROVED** [review]
Nice. This is something I've wanted for a while now.

File: src/scripts/clipperUI/mainController.tsx (hunk 1 above)
Comment (mttwc) [reply src/scripts/clipperUI/mainController.tsx:261]: Good idea!

**Review by VishaalK — APPROVED** [review]
I also learned how to use clocks today. Thanks Matthew !

**Pull request merged**